docker/edge-alpine.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM alpine as base
```

Comment:
I'm curious why `alpine:latest` instead of picking a version? 
Thinking about the potential for a build to break in the future if anything in our image depends on something that should happen to change as alpine pushes forward over time.